language/tools/move-cli/README.md

```diff
@@ -35,8 +35,27 @@ you can find the complete list of commands available by calling `move
 for each Move CLI command can be found by passing the `--help` flag to it,
 i.e., `move <command> --help`.
 
+Move CLI commands are grouped into three large subcommands:
+* **base commands**: are basic Move commands such as compiling or checking a set of modules. These do not rely on a Move Adapter implementation nor implementation of storage.
+* **sandbox commands**: are commands that allow you to write Move modules and scripts, then run scripts and tests, and view the resulting state of execution in a local sandboxed environment.
+* **experimental commands**: are experimental commands that are currently in development.
 
-## Project structure
+## Base Commands
+
+You can use the `move check` command to compile and typecheck either a specific set of
```

Comment:
It might make sense to start with explaining `compile` here (since that's likely to be the most familiar/useful) and then say as an aside at the end that you can use `check` if you want to run the type checker, but not codegen.

Also: I personally favor a single `compile` command with a `--check` flag over multiple commands, but others may feel differently. In addition to a general preference for minimality, I think in time, we will have enough options to pass to the compiler that it will be annoying to partially duplicate them (e.g., we'll want `check` to have type system/linter options, but not codegen ones).

Reply:
Good point about sharing flags between compile and check. I would be fine with adding `--check` to compile and removing it as its own command. @tnowacki what do you think about this?